ibis/backends/bigquery/client.py

```diff
@@ -194,7 +194,53 @@ def bq_param_date(_: dt.Date, value, name):
 
 
 class BigQueryTable(ops.DatabaseTable):
-    pass
+    @property
+    def bq_table(self):
+        """get the google bigquery Table object for easy reference"""
+        return self.source.client.get_table(self.name)
```

Comment:
Should we be caching the properties that could take some time due to API calls? I'm thinking this one and `time_partitions` would be useful.

Reply:
Makes sense — I'm using the cached_property decorater, lmk if I should be using the previous version to keep it compatible with python versions below 3.8.